kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -298,11 +307,23 @@ protected Properties getBrokerProperties(int i) {
   @AfterEach
   public void tearDown() throws Exception {
     log.info("Starting teardown of {}", getClass().getSimpleName());
+    tearDownMethod();
+    log.info("Completed teardown of {}", getClass().getSimpleName());
```

Comment:
nit: You should remove the duplicating log at the end of the new `tearDownMethod()`, like you did in the `setupMethod()`.

Reply:
I've moved the log start and end lines to the methods doing the actual stopping, as they are called from more than one place - effectively entry and exit logs, which belong in the method doing the work, not the calling method.

I did this for the setup and tear down, and the stop rest call too.